lib/exqlite/sqlite3.ex

```diff
@@ -78,6 +78,27 @@ defmodule Exqlite.Sqlite3 do
   @spec step(db(), statement()) :: :done | :busy | {:row, []}
   def step(conn, statement), do: Sqlite3NIF.step(conn, statement)
 
+  @spec multi_step(db(), statement()) :: :busy | {:rows, [[]]} | {:done, [[]]}
+  def multi_step(conn, statement), do: multi_step(conn, statement, 50)
```

Comment:
can we make the chunk size configurable?

Reply:
I can see a higher chunk size being nice in environments where one is not concerned about startving out the dirty NIF scheduler (e.g., when there is a large core count)

Reply:
I think we could do that. Although for ecto_sqlite3, I'd like that to be configurable per connection and not just for all connections.

Reply:
For per-connection, I'm not sure how that would look. The only way I can think of making it configurable like that is by hooking into `:configure` and having it per index, but not sure how that would be useful. What were you picturing?

Reply:
Sorry I meant to say per repo.

```elixir
  use Ecto.Repo,
    otp_app: :myapp,
    adapter: Ecto.Adapters.SQLite3,
    chunk_size: 1000
```

stuff like that.

Reply:
got it, that makes sense. that's what I was thinking, too :+1: